services/age_service.js

```diff
@@ -0,0 +1,65 @@
+const ValidationError = require(`../helpers/ValidationError.js`)
+
+//gets the dob request, calculate and return age
+const calaculateAge = (dob) => {
+  //check if dob is passed
+  if (!dob) throw new ValidationError(`Please pass date of birth to continue`)
+  // try pass dob
+  let dobformated = new Date(dob)
```

Comment:
Codacy has a fix for the issue: [Insert `;`](https://app.codacy.com/gh/phiiileo/validate-age/pullRequest?prid=9953536)
```suggestion
  let dobformated = new Date(dob);
```